Core/Object Arts/Dolphin/MVP/Presenters/List/ListBox.cls

```diff
@@ -17,6 +17,16 @@ Class Variables:
 !ListBox categoriesForClass!MVP-Views! !
 !ListBox methodsFor!
 
+anchorIndex
```

Comment:
I remember coming up with the term "anchor" to describe the concept, early on when I was investigating the behavior of the control, before even looking at the documentation and finding the term "selection mark". So yes, I find it preferable also.

I was also vaguely wondering, as I made the changes, whether it would be worth standardizing, but for whatever reason didn't do so. Since you independently thought of the same idea, I gave it a shot, and it turns out to be totally worth it—it becomes possible to push up the `#state` changes to ListControlView, which saves a lot of code. (I do believe the ListBox "anchor" and ListView "selection mark" behave the same, yes.)